src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -280,11 +280,11 @@ paths:
           description: An image with the provided FIF was not found.
 
 # IIIF Image request
-  /{identifier}/:
+  /{identifier}/{region}/{size}/{rotation}/{quality}.{format}:
     get:
       tags:
-        - 'IIF Image request'
-      summary: 'OpenAPI definition for IIIF Image Request API '
+        - 'Access'
+      summary: 'IIIF Image Request'
       operationId: IIIFImageRequest
```

Comment:
The description says "Special characters must be URI encoded.". Maybe add a ", such as / and ?" and mention that the OpenAPI GUI will do it automatically if that is used.

Reply:
Description is now updated